CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 3.3.2 (2020-03-16)
+
+- Fixed the autocomplete component to properly highlight results after removing search text. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
+- Fixed the autocomplete component to properly highlight results when only one character is supplied. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
+- Fixed the country field component to properly fire the `selectedCountryChange` event when used outside of a form. [#63](https://github.com/blackbaud/skyux-lookup/pull/63)
+- Fixed the lookup component to interact with template driven forms correctly. [#64](https://github.com/blackbaud/skyux-lookup/pull/64)
```

Comment:
> ... to interact with template driven forms correctly.
Should we be more specific with what exactly was fixed here?

Reply:
I've changed that entry to match what the original issue was for. There might have been some other odd template form stuff but that should cover the main thing. @Blackbaud-DeniseCisneros I've reworded this entry a lot so it could use another look.